### DISCUSSION TITLE
Where is the trash?

### DISCUSSION THREAD

**Discussion by hrrrps** [doublecmd/doublecmd#1256] [General]
I am using Windows 11. I just deleted something that I would like to restore. How do I find trash? I checked Windows recycle bin, but nothing there.

**Comment by JonnyTech** [comment]
How exactly did you delete?

**Reply by hrrrps** [reply]
Good question. From within the program. Using a right-click menu option I think.

**Comment by hrrrps** [comment]
> How exactly did you delete?
Actually no, I used the keyboard button. When you delete from the right click menu it send the item to recycling bin in Windows. When you use the keyboard Delete DC asks if you want to delete into the trash can.

**Comment by alexx2000** [comment]
https://doublecmd.h1n.ru/viewtopic.php?p=40984
